tests/pytorch/test_torch_save_load.py

```diff
@@ -232,3 +241,21 @@ def test_fp8_model_checkpoint(
     with te.fp8_autocast():
         y = model(x.detach().clone())
         torch.testing.assert_close(y, y_ref, **tols)
+    
+    if load_fp8_model:
+        # [ This is used to test the scenario where save_fp8_model is set to False and load_fp8_model is set to True ]
```

Comment:
"This is part of logic that tests `save_fp8_model=False` and `load_fp8_model=True`"